xmodule/tests/xml/factories.py

```diff
@@ -58,7 +58,7 @@ def __repr__(self):
 
 # Extract all argument names used to construct XmlImportData objects,
 # so that the factory doesn't treat them as XML attributes
-XML_IMPORT_ARGS = inspect.getargspec(XmlImportData.__init__).args  # lint-amnesty, pylint: disable=deprecated-method
+XML_IMPORT_ARGS = inspect.getfullargspec(XmlImportData.__init__).args  # lint-amnesty, pylint: disable=deprecated-method
```

Comment:
If the pylint warning was coming up due to the `getargspec` method, then perhaps the comment can be removed now as well.